openssl-dynamic/src/main/c/sslutils.c

```diff
@@ -555,8 +554,7 @@ int SSL_CTX_use_certificate_chain(SSL_CTX *ctx, const char *file,
     return n;
 }
 
-int SSL_CTX_use_certificate_chain_bio(SSL_CTX *ctx, BIO *bio,
-                                  int skipfirst)
+static int SSL_CTX_setup_certs(SSL_CTX *ctx, BIO *bio, int skipfirst, int ca)
```

Comment:
I prefer to be explicit about types when possible in typed languages. If its an `int` then compare it to an explicit value, if its a `bool` then use it like a boolean. For example below we use the `int` like a boolean and there is nothing that describes this assumption ... making it a `bool` removes the ambiguity.